src/utils/DockerUtil.js

```diff
@@ -159,6 +160,12 @@ var DockerUtil = {
         containerData.HostConfig.PublishAllPorts = true;
       }
 
+      containerData.NetworkingConfig = {
+        EndpointsConfig: _.mapObject(containerData.NetworkSettings.Networks, (network) => {
```

Comment:
Also this will not work with multiple networks (https://github.com/docker/docker/issues/29265), I will trying to solve it.